lib/setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+from setuptools import setup, find_packages
+
+setup(
+    name='renewablepotentialslib',
+    version='0.1.0', # additionally defined in __init__.py
+    description='Library code of the solar-and-wind-potentials workflow.',
+    maintainer='calliope-project',
+    maintainer_email='bryn.pickering@usys.ethz.ch',
+    packages=find_packages(exclude=['tests*']),
```

Comment:
I disagree with _not_ defining dependencies here. These libraries _are_ dependencies, whether you like it or not. This has nothing to do with the way dependencies are installed (I agree with you: we prefer conda-forge).

Reply:
I initially did have all the dependencies, but then with everything breaking when trying to install the package via pip, I took them out. In a way what this does is force a user to install the package using conda and it removes the duplication of defining dependencies both here and in the various conda requirements files. However, I don't feel strongly about it, so can add them back in.

Reply:
I feel strongly about it. :D